_data/chains/eip155-164.json

```diff
@@ -11,24 +11,14 @@
     "decimals": 18
   },
   "infoURL": "https://docs.omni.network",
-  "shortName": "omni_testnet",
+  "shortName": "omni_omega",
   "chainId": 164,
   "networkId": 164,
   "slip44": 1,
   "explorers": [
-    {
-      "name": "Omni X-Explorer",
-      "url": "https://explorer.testnet.omni.network",
-      "standard": "none"
-    },
-    {
-      "name": "Omni EVM Explorer on Blockscout",
-      "url": "https://omni-testnet.blockscout.com",
-      "standard": "EIP3091"
-    },
     {
       "name": "Omni EVM Explorer on Routescan",
```

Comment:
Suggest "Omni EVM and cross-chain Explorer"

Reply:
Done